### DISCUSSION TITLE
Manifest V3

### DISCUSSION THREAD

**Discussion by katastrophe92** [uBlockOrigin/uBOL-home#76] [Q&A] [answered]
Hey! I only use Firefox and LibreWolf but I would like to use Brave as a daily driver. Does uBlock work on chrome browsers like it does on Firefox, now with the new Manifest V3? Thanks

**Comment by ghost** [comment, marked as answer]
For now, both MV2 and MV3 versions of uBO work on Chrome.

- MV2 https://chrome.google.com/webstore/detail/ublock-origin/cjpalhdlnbpafiamejdnhcphjbkeiagm
- MV3 https://chrome.google.com/webstore/detail/ublock-origin-lite/ddkjiahejlhfcafbddmgiahcphecmpfh
